plugins/base/src/main/kotlin/translators/psi/DefaultPsiToDocumentableTranslator.kt

```diff
@@ -421,25 +421,27 @@ object DefaultPsiToDocumentableTranslator : SourceToDocumentableTranslator {
         private fun PsiAnnotationMemberValue.toValue(): AnnotationParameterValue? = when (this) {
             is PsiAnnotation -> toAnnotation()?.let { AnnotationValue(it) }
             is PsiArrayInitializerMemberValue -> ArrayValue(initializers.mapNotNull { it.toValue() })
-            is PsiReferenceExpression -> driOfReference()?.let { EnumValue(text ?: "", it) }
+            is PsiReferenceExpression -> psiReference?.let { EnumValue(text ?: "", DRI.from(it)) }
             is PsiClassObjectAccessExpression -> ClassValue(
                 text ?: "",
                 DRI.from(((type as PsiImmediateClassType).parameters.single() as PsiClassReferenceType).resolve()!!)
             )
             else -> StringValue(text ?: "")
         }
 
-        private fun PsiAnnotation.toAnnotation() = driOfReference()?.let {
+        private fun PsiAnnotation.toAnnotation() = psiReference?.let {
```

Comment:
You should really avoid having multiple "it"'s in scope. This makes it unnecessarily hard to read while giving you almost 0 benefit while writing!

```kotlin
private fun PsiAnnotation.toAnnotation() = psiReference?.let { psiReference -> 
```

Reply:
Yeah, this PR is quite old. Recently I started giving names

Reply:
Will change